### PR TITLE
Add methods to construct Coll from Java and use in ErgoValue more easy

### PR DESCRIPTION
**Pull request by MrStahlfelge**
This adds method to use `Coll` more easily from Java

### CODE REVIEW

File: appkit/src/test/scala/org/ergoplatform/appkit/ErgoValueTest.java [hunk 1]
```diff
@@ -34,7 +42,11 @@ public void testTypeDeclarations() {
         assertEquals(true, booleanValue);
         assertEquals(1, byteValue);
         assertEquals(1, shortValue);
+        assertFalse(booleanFromCollValue);
         assertEquals(0, byteFromCollValue);
+        assertEquals(0, shortFromCollValue);
+        assertEquals(0, intFromCollValue);
+        assertEquals(0, longFromCollValue);
```
Comment (aslesarenko) [inline appkit/src/test/scala/org/ergoplatform/appkit/ErgoValueTest.java:49]: I think `0` is not good test vector value, as it often may mean `uninitialized` value.

File: common/src/main/java/org/ergoplatform/appkit/JavaHelpers.scala [hunk 2]
```diff
@@ -475,6 +475,14 @@ object JavaHelpers {
...
   }
 
+  def collFrom(arr: Array[Long]): Coll[Long] = Colls.fromArray(arr)
+
+  def collFrom(arr: Array[Int]): Coll[Int] = Colls.fromArray(arr)
+
+  def collFrom(arr: Array[Boolean]): Coll[Boolean] = Colls.fromArray(arr)
+
+  def collFrom(arr: Array[Short]): Coll[Short] = Colls.fromArray(arr)
+
```
Comment (aslesarenko) [inline common/src/main/java/org/ergoplatform/appkit/JavaHelpers.scala:485]: Of course it is better to have Iso[Array[T], Coll[T]], but this is also good. :)

**Review by aslesarenko — APPROVED** [review]
Looks good, but please see comments.

File: common/src/main/java/org/ergoplatform/appkit/JavaHelpers.scala (hunk 2 above)
Comment (MrStahlfelge) [reply common/src/main/java/org/ergoplatform/appkit/JavaHelpers.scala:485]: The iso is not vital as from Java values of Coll can be accessed with coll.apply(i) - but it is not possible to construct a `Coll`

**Pull request merged**